azure-pipelines.yml

```diff
@@ -108,6 +108,8 @@ stages:
             python.version: '3.8'
           win-Python39:
             python.version: '3.9'
+          win-Python310:
+            python.version: '3.10'
```

Comment:
7 lines above, there is still Python version 3.6 - good moment to remove?

Reply:
probably a good idea, in that we don't even test it against 3.6 anymore